modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+// * hosted-control-planes/index.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-concepts-personas_{context}"]
+= Glossary of common concepts and personas for hosted control planes
+
+When you use hosted control planes for {product-title}, it is important to understand its key concepts and the personas that are involved.
+
+[id="hosted-control-planes-concepts_{context}"]
+== Concepts
+
+hosted cluster:: An {product-title} API endpoint for the tenant cluster that is managed by the HyperShift Operator.
+
+hosted cluster infrastructure:: Network, compute, and storage resources that exist in the tenant or end-user cloud account.
+
+hosting cluster:: See _management cluster_.
+
+hosted control plane:: An {product-title} control plane that runs on the management cluster, which is exposed by the API endpoint of a hosted cluster. The components of a control plane include etcd, an API server, the Kubernetes controller manager, and a VPN.
+
+management cluster:: An {product-title} cluster where the HyperShift Operator is deployed and where the control planes for tenant clusters are hosted. In some contexts, the management cluster is referred to as the _hosting cluster_.
+
+management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
+
+[id="hosted-control-planes-personas_{context}"]
+== Personas
+
+cluster instance administrator:: The role that is assumed by the user who is the equivalent to the administrator in stand-alone {product-title}. This user has the cluster-admin role in the provisioned cluster, but might not have power over when or how the cluster is updated or configured. This user might have read-only access to see some configuration projected into the cluster.
+
+cluster instance user:: The role that is assumed by the user who is the equivalent of a developer in stand-alone {product-title}. This user does not have a view into OperatorHub or machines.
+
+cluster service consumer:: Users who assume this role can request control planes and workers, drive updates, or modify externalized configurations. Typically, this user does not manage or access cloud credentials or infrastructure encryption keys. The cluster service consumer persona can request hosted clusters and interact with node pools. Users who assume this role have RBAC to create, read, update, or delete hosted clusters and node pools within a logical boundary.
```

Comment:
I don't think we normally say just "workers". It would usually be "control plane nodes and worker nodes", if that's what you mean here